torch_xla/csrc/lowering_context.h

```diff
@@ -68,7 +72,8 @@ class LoweringContext : public torch::lazy::LoweringContext {
   xla::XlaOp GetOutputOp(const torch::lazy::Output& output);
 
   // Build the XLA computation capturing all the operations created with the
-  // embedded XLA builder (returned by the builder() API).
+  // embedded XLA builder (returned by the builder() API) with check whether
```

Comment:
Think we can keep the original comment. Add the while specific comment on the new private method, as described above.

Reply:
Thanks, updated